### PR TITLE
(chore) O3-3372: Use React Form Engine in E2E Tests

### PR DESCRIPTION
**Pull request by jayasanka-sack**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary

The OpenMRS distribution and O3 environments are currently utilizing the React form engine. To ensure consistency and improve testing accuracy, we need to incorporate the React form engine (form-engine-app) in our E2E tests.

## Related Issue

https://openmrs.atlassian.net/browse/O3-3372

### CODE REVIEW

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @jayasanka-sack!

File: playwright.config.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ const config: PlaywrightTestConfig = {
   use: {
     baseURL: `${process.env.E2E_BASE_URL}/spa/`,
     storageState: 'e2e/storageState.json',
-    video: 'retain-on-failure',
+    video: 'on',
```
Comment (ibacher) [inline playwright.config.ts:22]: Did we mean to make this change permanent?

Comment (jayasanka-sack) [reply playwright.config.ts:22]: Oh!! my bad it was meant to be temporary.

Comment (jayasanka-sack) [reply playwright.config.ts:22]: fixed with https://github.com/openmrs/openmrs-esm-patient-chart/commit/3b7377d5507f7d5e1a8e0fbc849ed4ff97d670cd

**Pull request merged**